test/connectors/test_torch_connector.py

```diff
@@ -44,6 +45,24 @@ def setup_test(self):
             torch.tensor([[[1.0], [2.0]], [[3.0], [4.0]]]),
         ]
 
+    def test_get_einsum_signature(self):
```

Comment:
Unit tests should really (best practice) test the public API and not implementation specific private methods etc that are an implementation detail. Such methods should be covered of course overall by unit tests which may mean calling the public API with certain values etc. In this case can the function working be covered by using the connector with suitably sized data and making sure a result is returned as expected?

Reply:
Doing that is possible. Would you advise rebuilding the test using something similar to https://github.com/qiskit-community/qiskit-machine-learning/issues/716#issuecomment-1990945075?

Reply:
I agree with @woodsp-ibm. I think the best way is exactly to have a test like in the issue. If possible, I'd generalize it.